oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleProvider.java

```diff
@@ -355,18 +355,20 @@ public void start() throws ModuleStartException {
 
         Address gRPCServerInstanceAddress = new Address(moduleConfig.getGRPCHost(), moduleConfig.getGRPCPort(), true);
         TelemetryRelatedContext.INSTANCE.setId(gRPCServerInstanceAddress.toString());
+        ClusterCoordinator coordinator = this.getManager()
+                                             .find(ClusterModule.NAME)
+                                             .provider()
+                                             .getService(ClusterCoordinator.class);
+        coordinator.startCoordinator();
+        coordinator.registerWatcher(remoteClientManager);
```

Comment:
Should we do register first before start coodinator?

Reply:
Not necessary, but do register first more clearly. I'll fix it